com.musala.simple.smarthome/com/musala/simple/smarthome/ISmartDevice.java

```diff
@@ -0,0 +1,6 @@
+package com.musala.simple.smarthome;
+
+public interface ISmartDevice {
+	
+	public String performAction();
```

Comment:
Please add javaDoc to this method. By convention, javaDoc has to be applied to all interface methods and CAN be omitted for overriden methods.